examples/okta_brand/README.md

```diff
@@ -0,0 +1,11 @@
+# okta_brand
+
+This resource represents a brand for an Okta organization. For more information
```

Comment:
Nit: For more information can be found -> More information can be found

Reply:
👍